pkg/lib/cockpit.d.ts

```diff
@@ -257,9 +257,16 @@ declare module 'cockpit' {
         remove(): void;
     }
 
+    interface ReplaceAttrs {
```

Comment:
At least `mode` should also get added while you're at it.  I think.  I'm not 100% sure what we should do with this and its interaction with umask...

`size` is also in the fsinfo blob.  This would be been an interesting alternative to the size-hint attribute we added at the top level.  I think maybe the ship has sailed on that one already?